docs/client-libraries-client-objects.md

```diff
@@ -0,0 +1,24 @@
+---
+id: client-libraries-client-objects
+title: Work with client object
```

Comment:
```suggestion
title: Work with client
```

How about removing `object` to make it consistent with other topics like `work with producer`?

Reply:
Done.